db/db_setup.sql

```diff
@@ -1,7 +1,7 @@
 
 -- folksonomy main table, with public/private partitionning
 CREATE TABLE folksonomy (
-    product     varchar(13)   NOT NULL,
+    product     varchar       NOT NULL,
```

Comment:
I'm not in favor of no limit at all.
We should also have limites for k/v to avoid getting garbage too easily.

Reply:
@cquest do let me know the safe limit that is required, so that I can change the regex and table definition accordingly.

Reply:
No idea, I would say less than 20

Reply:
I think there is no improvement by adding a constraint to VARCHAR actually according to this [link](https://stackoverflow.com/questions/1067061/does-a-varchar-fields-declared-size-have-any-impact-in-postgresql). But, there might be some performance improvements by shifting to the TEXT data-type.

So, do let me know if you want me to change the data-type and a suitable limit for the same.

Reply:
@aadarsh-ram we spoke of a limit of 24.

This is not for perfs issues but more to avoid having strange data in the database.

Reply:
@alexgarel Added a limit of 24.